scripts/carrot/HC_carrot_eval_default_inputs.json

```diff
@@ -0,0 +1,69 @@
+{
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_evalVcf":"test_output:VariantCallingCarrotOrchestrated.NIST_output_vcf",
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_evalLabel":"TESTSNAPSHOT2018HG002",
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_evalVcfIndex":"test_output:VariantCallingCarrotOrchestrated.NIST_output_vcf_index",
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_evalMonitoringExample": "test_output:VariantCallingCarrotOrchestrated.NIST_representative_benchmarking",
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_controlVcf":"test_output:VariantCallingCarrotOrchestrated.NIST_control_vcf",
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_controlLabel":"CONTROLSNAPSHOT2018HG002",
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_controlVcfIndex":"test_output:VariantCallingCarrotOrchestrated.NIST_control_vcf_index",
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_controlMonitoringExample": "test_output:VariantCallingCarrotOrchestrated.NIST_control_representative_benchmarking",
+
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_evalRuntimeSummaries": "test_output:VariantCallingCarrotOrchestrated.NIST_output_runtimes",
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_controlRuntimeSummaries": "test_output:VariantCallingCarrotOrchestrated.NIST_control_output_runtimes",
+
+  "BenchmarkVCFsHeadToHeadOrchestrated.NIST_truthVcf":"gs://dsp-methods-carrot-data/test_data/haplotypecaller_tests/HG002_GRCh38_GIAB_1_22_v4.2.1_benchmark.broad-header.vcf.gz",
```

Comment:
This also could maybe go in a gatk-related bucket.  Ideally, I think we'd have at least most of the stuff for these tests in the same bucket/directory/whatever, but we can also maybe worry about that once we get the branch-vs-master stuff in, if that's going to replace this.